docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/is-amount-of-assets-small.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Is amount of assets small (images, assets manager, videos, import data (if needed) etc.) Files: <30 GB?
```

Comment:
1. what is assets manager? 
2. Why is 30GB only in the title, but not in the text? What do we do if it's more than 30GB?

Reply:
According to someone's migration experience, it takes ~1pd to migrate assets if the amount is less than 30 GB and ~2pd if the amount is more than 30 GB. So looks like the formula in this document is not full, unfortunately.

Reply:
What about the question 1?